alma/common/install_mpis.sh

```diff
@@ -0,0 +1,103 @@
+$COMMON_DIR/download_and_verify.sh $IMPI_2021_DOWNLOAD_URL "e85db63788c434d43c1378e5e2bf7927a75d11aee8e6b78ee0d933da920977a6"
+bash l_mpi_oneapi_p_2021.6.0.602_offline.sh -s -a -s --eula accept
+mv ${INSTALL_PREFIX}/intel/oneapi/mpi/${IMPI_2021_VERSION}/modulefiles/mpi ${INSTALL_PREFIX}/intel/oneapi/mpi/${IMPI_2021_VERSION}/modulefiles/impi
+$COMMON_DIR/write_component_version.sh "IMPI_2021" ${IMPI_2021_VERSION}
+
+# Setup module files for MPIs
+mkdir -p /usr/share/Modules/modulefiles/mpi/
+
+# MVAPICH2
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/mvapich2-${MV2_VERSION}
+#%Module 1.0
+#
+#  MVAPICH2 ${MV2_VERSION}
+#
+conflict        mpi
+module load ${GCC_VERSION}
+prepend-path    PATH            /opt/mvapich2-${MV2_VERSION}/bin
+prepend-path    LD_LIBRARY_PATH /opt/mvapich2-${MV2_VERSION}/lib
+prepend-path    MANPATH         /opt/mvapich2-${MV2_VERSION}/share/man
+setenv          MPI_BIN         /opt/mvapich2-${MV2_VERSION}/bin
+setenv          MPI_INCLUDE     /opt/mvapich2-${MV2_VERSION}/include
+setenv          MPI_LIB         /opt/mvapich2-${MV2_VERSION}/lib
+setenv          MPI_MAN         /opt/mvapich2-${MV2_VERSION}/share/man
+setenv          MPI_HOME        /opt/mvapich2-${MV2_VERSION}
+EOF
+
+# OpenMPI
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/openmpi-${OMPI_VERSION}
+#%Module 1.0
+#
+#  OpenMPI ${OMPI_VERSION}
+#
+conflict        mpi
+module load ${GCC_VERSION}
+prepend-path    PATH            /opt/openmpi-${OMPI_VERSION}/bin
+prepend-path    LD_LIBRARY_PATH /opt/openmpi-${OMPI_VERSION}/lib
+prepend-path    MANPATH         /opt/openmpi-${OMPI_VERSION}/share/man
+setenv          MPI_BIN         /opt/openmpi-${OMPI_VERSION}/bin
+setenv          MPI_INCLUDE     /opt/openmpi-${OMPI_VERSION}/include
+setenv          MPI_LIB         /opt/openmpi-${OMPI_VERSION}/lib
+setenv          MPI_MAN         /opt/openmpi-${OMPI_VERSION}/share/man
+setenv          MPI_HOME        /opt/openmpi-${OMPI_VERSION}
+EOF
+
+#IntelMPI-v2021
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/impi_${IMPI_2021_VERSION}
+#%Module 1.0
+#
+#  Intel MPI ${IMPI_2021_VERSION}
+#
+conflict        mpi
+module load /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}/modulefiles/impi
+setenv          MPI_BIN         /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}/bin
+setenv          MPI_INCLUDE     /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}/include
+setenv          MPI_LIB         /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}/lib
+setenv          MPI_MAN         /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}/man
+setenv          MPI_HOME        /opt/intel/oneapi/mpi/${IMPI_2021_VERSION}
+EOF
+
+# Create symlinks for modulefiles
+ln -s /usr/share/Modules/modulefiles/mpi/mvapich2-${MV2_VERSION} /usr/share/Modules/modulefiles/mpi/mvapich2
+ln -s /usr/share/Modules/modulefiles/mpi/openmpi-${OMPI_VERSION} /usr/share/Modules/modulefiles/mpi/openmpi
+ln -s /usr/share/Modules/modulefiles/mpi/impi_${IMPI_2021_VERSION} /usr/share/Modules/modulefiles/mpi/impi-2021
```

Comment:
There are multiple install_mpis.sh scripts in different locations. I'd like to differentiate them through names.